### PR TITLE
React to break change in StringValues

### PR DESCRIPTION
**Pull request by rynowak**

### CODE REVIEW

File: src/Microsoft.AspNet.Authentication.Cookies/ChunkingCookieManager.cs [hunk 1]
```diff
@@ -87,7 +87,7 @@ public string GetRequestCookie(HttpContext context, string key)
                 for (var chunkId = 1; chunkId <= chunksCount; chunkId++)
                 {
                     var chunk = requestCookies[key + "C" + chunkId.ToString(CultureInfo.InvariantCulture)];
-                    if (chunk == null)
```
Comment (rynowak) [inline src/Microsoft.AspNet.Authentication.Cookies/ChunkingCookieManager.cs]: Oops - tests caught this one. This should be `Count == 0`

**Comment by Tratcher** [comment]
:shipit: when you fix it.

**Comment by rynowak** [comment]
Looks like this is already fixed. Thanks @HaoK

**Comment by HaoK** [comment]
No I mistakenly updated something else thinking it was related.  We still need this, looks good :shipit:

**Comment by rynowak** [comment]
Ok, pushing it then.

**Pull request closed without merging**